source/documentation/deploying_services/use_a_custom_domain.erb

```diff
@@ -417,10 +417,10 @@ Server error, status code: 409, error code: 60016, message: An operation for ser
 This happens because you cannot do anything to a service instance while it's in a pending state. A CDN service instance stays pending until it detects the CNAME or ALIAS record. If this causes a problem for you, [contact support](https://www.cloud.service.gov.uk/support) to ask us to manually delete the pending instance.
```

Comment:
```suggestion
This happens because you cannot do anything to a service instance while it is in a pending state. A CDN service instance stays pending until it detects the CNAME or ALIAS record. If this causes a problem for you, [contact support](https://www.cloud.service.gov.uk/support) to ask us to manually delete the pending instance.
```